### DISCUSSION TITLE
Changed the Doc site image hasn't reflected

### DISCUSSION THREAD

**Discussion by Uchebuzz** [GitbookIO/community#782] [0. General bugs and questions]
Hi, Team.
I need help, I changed the doc site image but it is still showing the an emoji.... Any suggestion

Could it be that the image size didn't match the require one? Any suggestions?

**Comment by addisonschultz** [comment]
Are you still seeing this? Sometimes the site might be cached, and take some time to update - but I think it should be showing now!

**Reply by Uchebuzz** [reply]
Not updated yet.. How do I delete the cache from my end?

**Reply by addisonschultz** [reply]
Can you share a link to your published site? And What you’d be expecting to see?
